modules/network-observability-auth-multi-tenancy.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-auth-mutli-tenancy_{context}"]
+= Configure authorization and multi-tenancy
+Define `ClusterRole` and `ClusterRoleBinding`. The `netobserv-reader` `ClusterRole` enables multi-tenancy and allows individual user access, or group access, to the flows stored in Loki. You can create a YAML file to define these roles.  
+
+.Procedure
+
+. Using the web console, click the Import icon, *+*. 
```

Comment:
Maybe
```suggestion
. Using the web console, navigate to the *xyz* page and click the *+* icon to import your YAML file. 
```
or
```suggestion
. Using the web console, navigate to the *xyz* page and click the *+* icon. 
```
To leave the YAML off for the next step

Reply:
@opayne1 Good observation! I think that what's missing is that the YAML needs to be created in the `netobserv` project. Because that "+" is on the top of the toolbar, like this, so its accessible on any page. But when you create the YAML, you need to associate it with a project, I believe. @jotak is it correct that when you define the roles, you do it in the `netobserv` project?
![image](https://github.com/openshift/openshift-docs/assets/77501157/aac1aa52-8f5e-4fc3-a2a7-bdf3b88396c3)

Reply:
I checked with Joel and he says that the namespace customers can create the YAML in any namespace, it doesn't have to be netobserv. And also, the page that they're on when they create the YAML is not important either. Its just a matter of creating it and it will work from anywhere.